screenings.go

```diff
@@ -0,0 +1,142 @@
+package checkr
+
+import (
+	"log"
+	"sync"
+)
+
+// Screenings ...
+type Screenings struct {
+	SSNTrace                     *SSNTrace
+	SexOffenderSearch            *SexOffenderSearch
+	GlobalWatchListSearch        *GlobalWatchListSearch
+	NationalCriminalSearch       *NationalCriminalSearch
+	FederalCriminalSearch        *FederalCriminalSearch
+	CountryCriminalSearches      []*CountryCriminalSearch
+	StateCriminalSearch          []*StateCriminalSearch
+	MotorVehicleReport           *MotorVehicleReport
+	EducationVerification        *EducationVerification
+	EmploymentVerification       *EmploymentVerification
+	IdentityDocumentVerification *IdentityDocumentVerification
+}
+
+// GetScreenings returns all the screenings for a report
+func GetScreenings(report *Report, c *Client) *Screenings {
```

Comment:
So for this one, it's best if you actually tie it to `Report` since it's a report object and this needs to return an error.

Reply:
got it will do that